### PR TITLE
updates LICENSE.txt and removes maintainer

### PR DESCRIPTION
**Pull request by gabrielcsapo**
🎉 2017 it's a new year!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,6 @@
 ![kraken-js](public/krakenLogo.png)
 
-
-kraken.js
-=========
-
-Lead Maintainer: [Jean-Charles Sisk](https://github.com/jasisk)
```
Comment (jasisk) [inline README.md:7]: 🎉

Comment (gabrielcsapo) [reply README.md:7]: @jasisk the sentiment is that we shouldn’t have repo maintainers, it puts pressure on those maintainers. We are moving towards a generalist maintainer model!

Comment (jasisk) [reply README.md:7]: Oh, don't get me wrong: I'm all for it! 👍

**Comment by grawk** [comment]
👍

**Pull request merged**